extensions/sql-bindings/package.nls.json

```diff
@@ -0,0 +1,4 @@
+{
+	"%displayName%": "SQL Bindings",
+	"%description%": "Enables users to develop and publish azure functions with sql bindings"
```

Comment:
Azure Functions should probably be capitalized (but it would be good to run this by other people to see if there's any other wording changes we may want to do)

(check other places like README too)

Reply:
discussed with Drew on wording for both description and README